### DISCUSSION TITLE
[没什么意义的讨论]为什么B站动态的图片在浏览器和客户端里显示的情况不同呢

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#390] [一般讨论]
原动态[https://t.bilibili.com/754213613345964120](https://t.bilibili.com/754213613345964120)
这个是客户端里的情况
不是透明底的图有两张（除开中间那张
![Screenshot_2023-01-23-13-39-47-990_tv.danmaku.bili.jpg](https://user-images.githubusercontent.com/66856838/213973125-2de29cc5-8f2e-4eba-9f3a-c1acab61e007.jpg)

这个是机器人推送到群里的情况
不是透明底的图有四张
![IMG_20230123_134033.png](https://user-images.githubusercontent.com/66856838/213973211-f45483ea-ea92-4523-8a83-ff4c595ec1db.png)

这个是网页浏览的情况
不是透明底的图也是四张，和推送到群里的一样
![IMG_20230123_134104.png](https://user-images.githubusercontent.com/66856838/213973269-c6666feb-9bc1-4404-a8e9-230d1044d409.png)

**Comment by Tsuk1ko** [comment]
母鸡，B站预览图是它自己转的 webp 就透明了，原图 png 不透明，不知道B站怎么回事

https://i0.hdslb.com/bfs/new_dyn/84a98c15c938fa5bd554300f00963b7b1428441528.png

https://i0.hdslb.com/bfs/new_dyn/84a98c15c938fa5bd554300f00963b7b1428441528.png@1036w.webp
